source/mainnet/net/guides/create-proofs.rst

```diff
@@ -0,0 +1,148 @@
+.. include:: ../../variables.rst
+.. _create-proofs:
+
+=============
+Create proofs
+=============
+
+The |bw| supports secret proofs that allow dApps or services to request proof that the user meets some requirement, such as proof the user is over a certain age, or resides in a specific range of countries or area. The user can choose whether they want to reveal these :ref:`attributes<glossary-attribute>` to the dApp or service. The dApp or service uses a :ref:`zero knowledge proof<glossary-zero-knowledge-proof>` to request proof of the attribute(s) necessary without revealing anything beyond the truth of the statement. The dApp or service can also request that attributes are revealed. The wallet owner chooses whether to prove or reveal these :ref:`attributes<glossary-attribute>` to the dApp or service.
```

Comment:
Did we not agree to use the phrase "zero knowledge proof" instead of "secret proof"? The latter is not correct or meaningful.

Reply:
Additionally, this part is misleading to me
```
The user can choose whether they want to reveal these :ref:`attributes<glossary-attribute>` to the dApp or service. The dApp or service uses a :ref:`zero knowledge proof<glossary-zero-knowledge-proof>` to request proof of the attribute(s) necessary without revealing anything beyond the truth of the statement.
```

there are two different things here as I see it.

In option one the user can choose to **reveal** the attributes directly. I think it is misleading to talk about zero knowledge proofs in this case since the attribute is just revealed.

In the second case only the weaker statement is revealed (such as "Residence in EU"), and there is a zero knowledge proof included (zero knowledge means you only get to know that the country is in EU in this case).